tools/clang/unittests/HLSL/ValidationTest.cpp

```diff
@@ -1614,10 +1614,6 @@ TEST_F(ValidationTest, BigStructInBuffer) {
   TestCheck(L"..\\CodeGenHLSL\\BigStructInBuffer.hlsl");
 }
 
-TEST_F(ValidationTest, GloballyCoherent2) {
-  TestCheck(L"..\\CodeGenHLSL\\globallycoherent2.hlsl");
-}
-
 TEST_F(ValidationTest, GloballyCoherent3) {
   TestCheck(L"..\\CodeGenHLSL\\globallycoherent3.hlsl");
```

Comment:
globallycoherent3 tests for the same error message as globallycoherent2. It seems consistent to move both if we move one to VerifierTest